index.html

```diff
@@ -9,17 +9,22 @@
   <link rel="stylesheet" href="./CSS/style.css" />
 </head>
 <body>
-  <header class="main-header d-flex flex-center">
+  <header class="main-header d-flex">
     <h2>Welcome</h2>
-    <nav>
+    <nav class="d-flex">
+      <ul class="nav-menu d-flex flex-center">
+        <li><a href="#">Portoflio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
```

Comment:
To make the transition between these links and the sections, you should add the id section in the `href` value for the links, for instance:
```
<a href="#section-home">Home</a> //Link to go to home section, remember use the # before the id section.

<section id="section-home">
  <div>
  </div>
</section>
```

- [ ] [OPTIONAL] Make the transition between your nav links and the sections.